<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title1_LB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>课程信息与IP设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="lesson_name_LB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>课程名称</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lesson_name_TEXT">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ip_address_LB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP地址</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_address_TEXT">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="port_num_LB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_num_TEXT">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="stu_list_src_LB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>学生名单</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="stu_list_src_TEXT">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="set_up_ip_BTN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>设置完成</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="stu_sign_in_TB">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="print_absent_BTN">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>显示未签到学生</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="stu_num_LCD">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="all_message_TB">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>491</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="send_msg_TEXT">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send_msg_BTN">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_jpg_BTN">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_file_BTN">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_test_BTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开启小测</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
